tests/test_infected_asyncio.py

```diff
@@ -170,11 +170,11 @@ async def trio_ctx(
     # message.
     with trio.fail_after(2):
         async with (
+            trio.open_nursery() as n,
```

Comment:
Just making a mental note of this change: obviously it makes sense for this test to hang if the cancellation from the `.open_channel_from()` never propagates bc the nursery was still sleeping.